src/lib-sana/Graph/Graph.cpp

```diff
@@ -71,3 +110,41 @@ void Graph::ClearGraph() {
     adjLists.clear();
     numNodes = numEdges = 0;
 }
+
+void Graph::setGeneCount(uint geneCount) {

```

Comment:
Not all graphs are Gene, MiRNA type. Originally this part was done in a hurry so things related to Gene and MiRNA might change from 1.0 version. Keeping it for now.